components/tag-list/tag-list-item-mixin.js

```diff
@@ -276,4 +277,16 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		`;
 	}
 
+	_validatePlainText() {
+		clearTimeout(this._validatingPlainTextTimeout);
+		// don't error immediately in case it doesn't get set immediately
+		this._validatingPlainTextTimeout = setTimeout(() => {
+			this._validatingPlainTextTimeout = null;
+			const hasPlainText = (typeof this._plainText === 'string') && this._plainText.length > 0;
```

Comment:
Just want to note that `typeof new String()` is `'object'`. Unlikely anyone is using it, but 🤷

Reply:
Fair, and that's ridiculous JavaScript.

Reply:
Ok updated.